authors/apps/authentication/account_activator.py

```diff
@@ -0,0 +1,35 @@
+import jwt
+from django.contrib.auth.models import User as SuperUser
+from django.http import HttpResponse
+from rest_framework import exceptions
+
+from authors import settings
+from .models import User
+
+
+def activate(request, token):
+    try:
+        payload = jwt.decode(token, settings.SECRET_KEY)
```

Comment:
You can import the decoding method from JWTAuthentication in backend file. No need to repeat the code.